mmv1/templates/terraform/examples/forwarding_rule_vpc_psc.tf.erb

```diff
@@ -0,0 +1,29 @@
+// Forwarding rule for VPC private service connect endpoint
+resource "google_compute_forwarding_rule" "<%= ctx[:primary_resource_id] %>" {
+  provider              = google-beta
+  name                  = "<%= ctx[:vars]['forwarding_rule_name'] %>"
+  region                = "us-central1"
+  load_balancing_scheme = ""
+  target                = "projects/SERVICE_PROJECT/regions/REGION/serviceAttachments/SERVICE_NAME"
```

Comment:
Will this value work in a test as-is? Or does it actually need to be set to a valid target to pass (accepted by the API and "created" but not necessarily functional)

Reply:
It's need to be set to a valid value, "as-is" will failed.

Reply:
For our examples we like to have them testable and tests self-contained when possible. Can the target be set up in this example itself?

Reply:
Ok, I will work on it